test/fixtures/output/configuration/help-onprem.golden

```diff
@@ -7,8 +7,8 @@ Aliases:
   configuration, config
 
 Available Commands:
-  list        List configuration fields in ~/.confluent/config.json.
-  set         Set a field's value in ~/.confluent/config.json.
+  set         Set a user-configurable field's value.
+  view        View user-configurable fields in ~/.confluent/config.json.
```

Comment:
`update` and `list`?

Reply:
I talked about this in the 1-pager, but I think `set` and `view` are more fitting command names that also align with `kubectl`. `list` in particular implies a user can have multiple of a resource, but they only have 1 config.

Reply:
Alignment with our other commands is more important, IMO!

> list in particular implies a user can have multiple of a resource, but they only have 1 config.

I think the configuration key=value pair is the resource in question here